conda_smithy/templates/build_steps.sh.tmpl

```diff
@@ -6,8 +6,13 @@
 # benefit from the improvement.
 
 set -xeuo pipefail
-export PYTHONUNBUFFERED=1
 export FEEDSTOCK_ROOT="${FEEDSTOCK_ROOT:-/home/conda/feedstock_root}"
+source ${FEEDSTOCK_ROOT}/.scripts/logging_utils.sh
+
+endgroup "Start Docker"
```

Comment:
Drone.io doesn't run the `run_docker_build.sh` script and so this call won't be matched. It is a no-op on drone so I think this is fine.